Circular_Queue.c

```diff
@@ -0,0 +1,76 @@
+
+
+#include "circular_Queue.h"
+
+void CircularQueue_Init(Queue_t *pQueue)
+{
+	pQueue->Front=0;
+	pQueue->Rear =0;
+	pQueue->Size =0;
+}
+
+
+void CircularQueue_Enqueue(Queue_t *pQueue , QUEUE_DATA_TYPE Data)
+{
+	pQueue->Rear = pQueue->Rear %QUEUE_SIZE;
```

Comment:
add parentheses to make the code more readable (to specify C Operator Precedence) e.g `(pQueue -> rear) % QUEUE_SIZE`